src/pipelines.js

```diff
@@ -736,7 +967,7 @@ class ObjectDetectionPipeline extends Pipeline {
         }
         images = await prepareImages(images);
 
-        let imageSizes = percentage ? null : images.map(x => [x.bitmap.width, x.bitmap.height]);
+        let imageSizes = percentage ? null : images.map((x) => [x.bitmap.width, x.bitmap.height]);
```

Comment:
(simply back to original)

```suggestion
        let imageSizes = percentage ? null : images.map(x => [x.bitmap.width, x.bitmap.height]);
```